pom.xml

```diff
@@ -84,6 +85,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
```

Comment:
???

Reply:
to set up intellij to do live hot swapping to debug on a test server had to add this